admin/banner/class-admin-banner-sidebar.php

```diff
@@ -82,30 +82,48 @@ protected function add_banner_spot( WPSEO_Admin_Banner_Spot $spot ) {
 	 * @return WPSEO_Admin_Banner_Spot
 	 */
 	protected function get_premium_spot() {
-		$premium_spot = new WPSEO_Admin_Banner_Spot( __( 'Premium', 'wordpress-seo' ), $this->banner_renderer );
-
-		$premium_spot->set_description(
+		$premium_spot = new WPSEO_Admin_Banner_Spot( '', $this->banner_renderer );
+
+		$premium_uri = WPSEO_Shortlinker::get( 'https://yoa.st/jj' );
+
+		$premium_spot->set_extra(
```

Comment:
We might need a new class for this in the future for rendering text banners: `WPSEO_Admin_Text_Banner ` that inherits from an interface `WPSEO_Admin_Banner ` and the current `WPSEO_Admin_Banner ` should be renamed to `WPSEO_Admin_Image_Banner`.